### PR TITLE
Resolved Issues #8 and #9

### PR DESCRIPTION
**Pull request by Jeffrey711**
Issue #8: Replaced JList with JTable.
Issue #9: Added ability to cd into directories using a listening button. Still need to display specified path.

### CODE REVIEW

**Comment by dcoffill** [comment]
Please rename W15_lab06 to W15_lab06.md, as per lab instructions.

**Comment by Jeffrey711** [comment]
I have renamed W15_lab06 to W15_lab06.md, as per lab instructions. Thanks!

File: src/edu/ucsb/cs56/projects/FTP_client/ClientGui.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 	private JPanel loginPanel;
 	private JPanel downloadPanel;
 	private JButton connectButton;
+    private JButton cdButton;
```
Comment (dcoffill) [inline src/edu/ucsb/cs56/projects/FTP_client/ClientGui.java]: Be careful about tabs vs spaces!  You've indented with spaces, but the rest of the project uses tabs, which can cause the code to look weird on other computers.

Comment (Jeffrey711) [inline src/edu/ucsb/cs56/projects/FTP_client/ClientGui.java]: It looks like IntelliJ defaults to using spaces when the Tab key is pressed. I have changed my settings to use the "tab character" in place of spaces now.

**Comment by rhelmot** [comment]
Merged manually.

**Pull request closed without merging**